app/stt_service.py

```
@@ -0,0 +1,121 @@
+                
+        except Exception as e:
+            return {
+                "error": f"STT 처리 중 오류 발생: {str(e)}",
+                "transcript": "",
+                "confidence": 0.0
+            }
+        finally:
+            # 임시 파일 정리
+            try:
+                os.unlink(tmp_file_path)
+            except:
```

Ruff (0.14.1)

Do not use bare `except`

(E722)